source/meta_frontmatter.ptx

```diff
@@ -191,7 +191,6 @@
         First, always be contributing. The majority of exercises in this book are designed to lead to direct and useful contributions to a project, no matter how small. Even a simple act, such as adding comments to a part of the code you don’t understand, can add real value to a project; that’s the great thing about community developed software. Contribution matters, and legitimate contributions, no matter how small, are welcome in many projects.
       </p>
       <p>
-        Second, ask for help when you’re stuck. If you have trouble with an exercise — and at some point you will — look to your fellow contributors for help. Your chosen project likely has mechanisms for getting in touch with the more advanced developers: mailing lists, IRC channels, Slack channels, Discourse channels, forums, or more than one of the above. Communicating with those around you is not only “not cheating,” it’s key to establishing greater understanding. Keep this in mind, though: in the real world, people are most likely to help those who are trying to help themselves. Before you ask someone a question of more senior developers, ask the same question of Google. A good rule of thumb: if you can’t figure something out in a few hours of searching the Internet, it’s reasonable to ask for a bit of help.
         Second, ask for help when you’re stuck. If you have trouble with an exercise — and at some point you will — look to your fellow contributors for help. Your chosen project likely has mechanisms for getting in touch with the more advanced developers: mailing lists, IRC channels, Slack channels, Discord channels, forums, or more than one of the above. Communicating with those around you is not only “not cheating,” it’s key to establishing greater understanding. Keep this in mind, though: in the real world, people are most likely to help those who are trying to help themselves. Before you ask someone a question of more senior developers, ask the same question of Google. A good rule of thumb: if you can’t figure something out in 15 minutes of searching the Internet, it’s reasonable to ask for a bit of help.
```

Comment:
The only two differences I could find between the two paragraphs were the following:
1. Discourse channels
   vs.
   Discord channels, and
2. if you can’t figure something out in a few hours of searching the Internet
   vs.
   if you can’t figure something out in 15 minutes of searching the Internet

In both cases I thought that you probably meant the latter, hence I deleted the first copy of the paragraph

Reply:
Great!  Thanks so much!